vyper/venom/effects.py

```diff
@@ -64,7 +64,7 @@ class Effects(Flag):
     "selfbalance": BALANCE,
     "extcodecopy": EXTCODE,
     "selfdestruct": BALANCE,  # may modify code, but after the transaction
-    "log": MEMORY,
+    "log": MEMORY | LOG,
```

Comment:
is this necessary? since `log` is already marked with the `LOG` write effect (which is stronger than read)